es-ds-components/components/es-verification-code.vue

```diff
@@ -0,0 +1,89 @@
+<script setup lang="ts">
+import InputOtp from 'primevue/inputotp';
+
+const model = defineModel<string>();
+const verificationCode = ref();
+
+const props = defineProps({
+    charCount: {
+        type: Number,
+        default: 5,
+    },
+});
+
+const count = ref(props.charCount);
+watch(model, (newVal) => {
+    verificationCode.value = newVal;
+});
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const updateCode = (newValue: any) => {
```

Comment:
Weird, I can see why you did this `any` here. I tried changing this type to string, but then `@update:model-value="updateCode"` complained that it was expecting a function that takes a boolean argument, which makes no sense, since this would come in as a string.

So that future generations don't get tripped up by this, can we add a comment here explaining that `@update:model-value` is erroneously requiring a `boolean` function argument, so we've worked around this by specifying `any`?